constantsGNO.py

```diff
@@ -0,0 +1,60 @@
+#it's better to keep token1 to ETH or WETH since it serves as a reference for prices
```

Comment:
does not apply. 
```suggestion
#it's better to keep token1 to the wrapped native token of the chain (e.g. WETH for ethereum and wxDAI for gnosis chain) since it serves as a reference for prices
```